services/src/main/java/org/fao/geonet/api/records/MetadataApi.java

```diff
@@ -540,7 +540,10 @@ public FeatureResponse getFeatureCatalog(
 
         RelatedResponse related = getRelated(metadataUuid, type, 0, 100, request);
 
-        if (related.getFcats() != null) {
+        if (related.getFcats() != null
```

Comment:
One can also define a helper method (NOT static), taking for example related.getFcats as parameter and returning a boolean whose name should enlighten a little, for example (I do not really understand the context) isAtLeastOneFacetDefined'.

Reply:
To find a name is the hardest part! :) Thank you for the suggestion